pkg/apis/agones/v1/apihooks.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package v1
```

Comment:
```suggestion

package v1
```

Nit: a line please between the copyright and the package - otherwise my IDE think it's a package comment. Also it's what we do everywhere else (actually surprised it passes linting!)